api/service.capnp

```diff
@@ -0,0 +1,54 @@
+using Go = import "/go.capnp";
+
+@0xfcba4f486a351ac3;
+
+$Go.package("discovery");
+$Go.import("github.com/wetware/ww/internal/api/service");
+
+
+interface ServiceDiscovery {
+    provider @0 (name :Text) -> (provider :Provider);
+    locator @1 (name :Text) -> (locator :Locator);
+}
+
+interface Provider {
+    provide @0 (location :SignedLocation) -> ();
+}
+
+interface Locator {
+    findProviders @0 (chan :Sender(SignedLocation)) -> ();
+    
+    using Sender = import "channel.capnp".Sender;
+}
+
+struct SignedLocation {
```

Comment:
I'm not a huge fan of the `Location` terminology.   How about `SignedRecord`?

Reply:
But what we are returning is a location, not a record